src/main.rs

```diff
@@ -19,6 +19,7 @@ use tokio::sync::Mutex;
 
 #[tokio::main]
 async fn main() -> miette::Result<()> {
+    miette::set_panic_hook();
```

Comment:
Print panics nicely with our error library. I might try to replace most uses of `miette` with `color-eyre` at some point because I like its output formatting better.